experiments/functional/zfs-LocalPV/zfspv-snapshot/test.yml

```diff
@@ -0,0 +1,158 @@
+    - /mnt/parameters.yml
+    
+  tasks:
+    - block:
+
+        ## Generating the testname for deployment
+        - include_tasks: /utils/fcm/create_testname.yml
+
+        ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: "/utils/fcm/update_litmus_result_resource.yml"
+          vars:
+            status: 'SOT'
+        
+        - block:
+            - name: Get the application pod name
+              shell: >
+                kubectl get pod -n {{ app_ns }} -l {{ app_label }} 
+                --no-headers -o custom-columns=:.metadata.name
+              args:
+                executable: /bin/bash
+              register: app_pod_name
+
+            - name: Check if the application pod is in running state
+              shell: >
+                kubectl get pods -n {{ app_ns }} -l {{ app_label }} 
+                --no-headers -o custom-columns=:.status.phase
+              args:
+                executable: /bin/bash
+              register: app_pod_status
+              failed_when: "'Running' not in app_pod_status.stdout"
+
+            - block:
+              - name: Create some test data into the application
+                include_tasks: "/utils/scm/applications/busybox/busybox_data_persistence.yml"
+                vars:
+                  status: 'LOAD'
+                  ns: "{{ app_ns }}"
+                  pod_name: "{{ app_pod_name.stdout }}"  
+              when: data_persistence == 'busybox'
+            
+            - block:
+              - name: Create some test data into the application
+                include_tasks: "/utils/scm/applications/mysql/mysql_data_persistence.yml"
+                vars:
+                  status: 'LOAD'
+                  ns: "{{ app_ns }}"
+                  pod_name: "{{ app_pod_name.stdout }}"  
+              when: data_persistence == 'mysql'
+                       
+            - name: Update the snapshot template with the test specific variables
+              template: 
+                src: snapshot.j2
+                dest: snapshot.yml
+
+            - name: create snapshot   
+              shell: >
+                kubectl create -f snapshot.yml
+              args:
+               executable: /bin/bash
+              register: status
+              failed_when: "status.rc != 0"
+
+            - name: Confirm that volumesnapshot {{ snapshot_name }} is ready to use
+              shell: >
+                kubectl get volumesnapshot.snapshot {{ snapshot_name }} -n {{ app_ns }} 
+                -o jsonpath='{.status.readyToUse}'
+              args:
+                executable: /bin/bash
+              register: isSnapshotReady
+              until: "isSnapshotReady.stdout == 'true'"
+              delay: 5
+              retries: 30
+
+            - name: Check the status for openebs resource for the created snapshot {{ snapshot_name }}
+              shell: >
+                kubectl get zfssnap -n {{ operator_ns }} -o jsonpath='{.items[].status.state}'
```

Comment:
we need to check for the particular snapshot

Reply:
Done !!